code-examples/quarkus/kafka-quickstart/pom.xml

```diff
@@ -1,23 +1,22 @@
 <?xml version="1.0"?>
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.rhoas</groupId>
-  <artifactId>rhoas-kafka</artifactId>
-  <version>1.0-${timestamp}</version>
+  <artifactId>rhoas-kafka-quickstart-example</artifactId>
+  <version>1.0-SNAPSHOT</version>
   <properties>
+    <kafka2.version>2.7.0</kafka2.version>
```

Comment:
We should be able to drop the kafka version override if you are moving to 1.13

Reply:
Done